salt/metalk8s/salt/master/files/salt-master-pod.yaml.j2

```diff
@@ -68,6 +72,38 @@ spec:
         - name: kubernetes-config
           mountPath: '/etc/kubernetes'
           readOnly: true
+    - name: salt-api
+      image: {{ build_image_name(salt_master_image, salt_master_version) }}
+      command: ['/tini']
+      args: ['salt-api', '--', '--log-level=info']
+      ports:
+        - name: api
+          containerPort: 4507
+          protocol: TCP
+      livenessProbe:
+        httpGet:
+          path: /
+          port: api
+{% if ip_candidates %}
```

Comment:
Instead of repeating the if/else block many times, you could define a variable and use it instead, no?

Reply:
That exactly the goal of `defaults` and `context` in `file.managed` state

Reply:
Fair enough, will do.